CODE_OF_CONDUCT.md

```diff
@@ -1,42 +1,53 @@
-# Contributor Covenant Code of Conduct
-
-## Our Pledge
-
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of experience,
-nationality, personal appearance, race, religion, or sexual identity and
-orientation.
-
-## Our Standards
-
-Examples of behavior that contributes to creating a positive environment
-include:
-
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
```

Comment:
I think it would be good to keep the above, at the very least in some form. It's not as dense as the rest of this text and it's also a neat summary.

Reply:
@StanczakDominik The part in "Our Pledge" section is equivalent to the part in the beginning of the pledge list below. Should I bring the "Our Standards" section back?

Reply:
I think it would not hurt to be a bit verbose and remind people of these things again. So I agree with @StanczakDominik  that we should keep them.